src/main/java/services/MeasurementValueReader.java

```diff
@@ -29,61 +17,45 @@
 import java.util.Date;
 
 
-/**
- * Die Klasse dient als Schnittstelle zur Datenhaltung der Messwerte.
- * Je nach konfiguriertem Protokoll wird die entsprechende Methode aufgerufen.
- *
- * Created by Nett on 04.06.2017.
- */
 public class MeasurementValueReader {
 
 
-    /**
-     * Dient als Weiche um die, gemaess Prokoll des Devices, passende Methode aufzurufen.
-     *
-     * @param deviceName
-     * @return
-     * @throws ReadWriteException
-     */
-    public static MeasurementValueXml getActualValue(String deviceName)throws ReadWriteException{
-
-        String protocol = DeviceMapperJson.getMeasurementValueProtocol(deviceName);
-        switch (protocol){
-
-            case "xml-1":
-                return getActualValueFromXml(deviceName);
-            case "txt-1":
-                return getActualValueFromTxt(deviceName);
-            case "xml-2":
-                return getActualValueFromXmlSax(deviceName);
-            default:
-                throw new ReadWriteException("Not possible to read protocol-type: " + protocol);
+    public static MeasurementValueXml getActualValue(String deviceName) {
+
+        try {
+            String protocol = DeviceMapperJson.getMeasurementValueProtocol(deviceName);
+
+            assert protocol != null;
+            switch (protocol){
+
+                case "xml-1":
+                    return getActualValueFromXml(deviceName);
+                case "txt-1":
+                    return getActualValueFromTxt(deviceName);
+                case "xml-2":
+                    return getActualValueFromXmlSax(deviceName);
+                default:
+                    throw new ReadWriteException("Not possible to read protocol-type: " + protocol);
+            }
+
+        }catch (ParserConfigurationException | IOException | SAXException e) {
```

Comment:
die exceptions sollten so lokal, d.h. so nahe am Erzeuger wie möglich, catched werden. das war vor deinen changes eher der fall ... :) 
weil: Jetzt hast du in einer abstrakten Methode getActualValue sehr technische Exceptions-Catches drin, die stark von der Implementierung der einzelnen Protokolle abhängig sind. Das ist nicht gut (eine Form von "mixed levels of abstraction" Code Smell, oder auch Verletzung von information hiding).